app/assets/javascripts/darkswarm/controllers/checkout/checkout_controller.js.coffee

```diff
@@ -9,9 +9,18 @@ Darkswarm.controller "CheckoutCtrl", ($scope, storage, Checkout, CurrentUser, Cu
   for field in $scope.fieldsToBind
     storage.bind $scope, "Checkout.order.#{field}",
       storeName: "#{prefix}_#{field}"
+
   storage.bind $scope, "Checkout.ship_address_same_as_billing",
     storeName: "#{prefix}_sameasbilling"
-    defaultValue: true
+    defaultValue: 'YES'
```

Comment:
Yes, there is. I have tried a lot. I found there was a bug to store the false value in the local storage. The false value sometimes can not be retrieved correctly, and can be affected by other form fields. I would like to show you next week with master branch.